roles/set_openstack_containers/templates/update_env_vars.sh.j2

```diff
@@ -10,6 +10,9 @@ export PATH="{{ cifmw_path }}"
 {#   Add `-` to the cifmw_set_openstack_containers_prefix to avoid issues with #}
 {#   openstack-openstackclient image name #}
 {% set _container_prefix = cifmw_set_openstack_containers_prefix + '-' -%}
+{% if cifmw_set_openstack_containers_openstack_version_change | bool %}
+{%     set _ = containers_dict.update({'OPENSTACK_RELEASE_VERSION': '0.0.2'}) -%}
```

Comment:
This should be a variable, and we should use the same that the one we use to populate `OPENSTACK_VERSION` in the [install_yaml review](https://github.com/openstack-k8s-operators/install_yamls/pull/843)  which should be set in the [ci framework review](https://github.com/openstack-k8s-operators/ci-framework/pull/1852).

Maybe `cifmw_update_openstack_target_version`  for all of them?

Reply:
This sounds reasonable, I'll change 0.0.2 to variable.

Reply:
done